custom/conf/app.example.ini

```
@@ -938,8 +938,8 @@ PATH =
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
-;; List of prefixes used in Pull Request title to mark them as Work In Progress
-;WORK_IN_PROGRESS_PREFIXES = WIP:,[WIP]
+;; List of prefixes used in Pull Request title to mark them as Work In Progress (matched in a case-insensitive manner)
+;WORK_IN_PROGRESS_PREFIXES = Draft:,[Draft],WIP:,[WIP]
```

You haven't changed the defaults in modules/setting/repository.go, therefore do not change the documented defaults. These have to match the real situation.

In reality I think changing the defaults would be a breaking change therefore we should not do that. So I'll write a suggestion and change them back.

```suggestion
;WORK_IN_PROGRESS_PREFIXES = WIP:,[WIP]
```

---

Yes, that's fine. I agree we should not change the defaults in a PR like this.